### ISSUE TITLE
Does not respond to click on checkbox' label

### ISSUE THREAD

**Issue by AlexeyKosov** [minhur/bootstrap-toggle#23]
It would make sense to change the state when clicking on the label of the checkbox

**Comment by mewsoft** [comment]
Yes I also confirm on this request, it is a must to complete the element functionality. It is a great plugin to bootstrap.

**Comment by minhur** [comment]
I don't plan on supporting this feature.

`<form>` tag's `reset` event triggers before the fields's `change` event, making it hard to listen in on the dom change event *after* the form has been reset. Refer to mdn documentation on reset event to learn more.

The only workaround I would suggest here would be create a custom reset button that does a form reset using js, and update the toggles after the form reset completes.

Here's an example: http://jsfiddle.net/minhur/p8qbuepj/

**Comment by nickpierpoint** [comment]
Has there been any progress with this issue? I see from the examples on the home page that clicking on the label still doesn't trigger a toggle:

```
<label class="checkbox-inline">
  <input type="checkbox" checked data-toggle="toggle"> First
</label>
<label class="checkbox-inline">
  <input type="checkbox" data-toggle="toggle"> Second
</label>
<label class="checkbox-inline">
  <input type="checkbox" data-toggle="toggle"> Third
</label>
```

**Comment by ollyg** [comment]
For those using google, this can be achieved using a simple onclick handler, based on the following:

````
<label>
  <input type="checkbox" name="mycheck" id="id_mycheck" data-toggle="toggle">
  <span onclick="$('#id_mycheck').bootstrapToggle('toggle')">&nbsp;My Check Toggle</span>
</label>
````
